index.js

```diff
@@ -287,21 +287,21 @@ function objEquiv(a, b, opts, channel) {
   var aIsError = a instanceof Error;
   var bIsError = b instanceof Error;
   if (aIsError !== bIsError) { return false; }
-  if (aIsError || bIsError) {
+  if (aIsError) {
```

Comment:
`aIsError == bIsError` from the check above, so we only need to check one.

Reply:
Per the comments, all of these redundant comparisons are quite intentional, because it conveys the correct behavior to readers more clearly. What’s the advantage of removing them?

Reply:
It's faster because you're doing less comparisons. The behavior is still clear to the reader as the `aIsError !== bIsError` is right above it.

Reply:
Do you have benchmarks to indicate that the extra truthiness comparison, only in the falsy cases, actually slows down a program in any measureable way?

As the author of the code, it is *not* clear to me without the repetition, which is precisely why I left them all there intentionally.

Reply:
No the improvement is so small that you probably won't see this in any benchmark. Behavior could also be explained in a comment instead of extra code.

Reply:
If it doesn't objectively improve clarity, and it doesn't improve performance, what is being improved?

Reply:
I would still argue it doesn't change the clarity and even if you can't see the difference in a benchmark is still an easy way to do less comparisons. But it's your repo and if you don't see it this way I'll close the pull request.

Reply:
Thanks for the contribution anyways!